vocab/vocab.pyx

```diff
@@ -85,42 +129,7 @@ cdef class Vocabulary:
                     ngram[token] = tokenid
                 v.push_back(ngram)
 
-        self._vocabptr = new Vocab(v)
-        self.counts = np.array(counts, dtype=np.uint32) if counts else None
-        self.stopwords = Stopwords()
-
-        if build_table and counts:
-            self._table = self._build_table(table_size, power)
-        else:
-            self._table = None
-
-    def _build_table(self, table_size, power):
-        """
-        Create a table using the vocabulary tokens counts, which can be used
-        for drawing random words in negative sampling routines (useful for
-        word2gauss.)
-        This function is called by the constructor if the build_table
-        flag is set to True.
-
-        From gensim's make_table in word2vec.py
-        """
-        table = np.zeros(table_size, dtype=np.uint32)
-        # compute sum of all power (Z in paper)
-        train_words_pow = float(sum((count**power for count in self.counts)))
-        # go through the whole table and fill it up with the word indexes
-        # proportional to a word's count**power
-        widx = 0
-        # normalize count^0.75 by Z
-        d1 = self.counts[widx]**power / train_words_pow
-        vocab_size = len(self.counts)
-        for tidx in xrange(table_size):
-            table[tidx] = widx
-            if 1.0 * tidx / table_size > d1:
-                widx += 1
-                d1 += self.counts[widx]**power / train_words_pow
-            if widx >= vocab_size:
-                widx = vocab_size - 1
-        return table
+        self._vocabptr = new Vocab(v, STOP_WORDS)
```

Comment:
It'd be awesome to allow the user to specify the stop word list if they didn't want to use the default list.